test/test.cpp

```diff
@@ -51,7 +49,6 @@ void report_failure(char const* err, char const* file, int line)
 	char buf[500];
 	std::snprintf(buf, sizeof(buf), "\x1b[41m***** %s:%d \"%s\" *****\x1b[0m\n", file, line, err);
 	std::fprintf(stderr, "\n%s\n", buf);
-	failure_strings.push_back(buf);
```

Comment:
I used to print all of these at the end. I take it that's not the case anymore? I think it's a pretty convenient feature, to potentially avoid digging through a bunch of log output